### PR TITLE
Update events.json

### PR DESCRIPTION
**Pull request by owenabbott**

### CODE REVIEW

File: app/config/temple/events.json [hunk 1]
```diff
@@ -1,51 +1,51 @@
 {
     "events": [
         {
-            "name": "Who are the tenets talking to? by Minister Penemue",
+            "name": "The Paradox and Dangers of Superstitions by Minister VRap",
```
Comment (LadyAbigor) [inline app/config/temple/events.json:4]: "name": "More than just TST by Minister Chalice Blythe",

**Review by LadyAbigor — CHANGES_REQUESTED** [review]
"name": "More than just TST by Minister Chalice Blythe",

File: app/config/temple/events.json (hunk 1 above)
Comment (LadyAbigor) [inline app/config/temple/events.json]: "name": "More than just TST by Minister Chalice Blythe",

**Review by LadyAbigor — DISMISSED** [review]

**Review by owenabbott — COMMENTED** [review]
Corrected title for temple 8/2

**Review by LadyAbigor — APPROVED** [review]

**Pull request merged**